file1.js

```diff
@@ -0,0 +1,181 @@
+
+      var $this = $(this);
+      var postVariable = {};
+      postVariable.code = $(".VoucherText").val();
+
+      var url = $this.data("url");
+      utils.JSend(url, postVariable, function(data) {
+          $(".VoucherFeedback").addSuccess("Promocode added. Your trial was extended.");
+          $(".VoucherText").val("");
+          $(".VoucherButton").prop("disabled", true);
+        }, function(error) {
+          $(".VoucherFeedback").showError(error);
+        },
+        "Could not add the promocode, please try again.");
+    });
+
+    $(".BtnGenerateToken").click(function() {
+      var $this = $(this);
+
+      var url = $this.data("url");
+      utils.JSend(url, {}, function(data) {
+          $(".ApiTokensFeedback").showSuccess("New token generated");
+          location.reload();
+        }, function(error) {
+          $this.blur();
+          $(".ApiTokensFeedback").showError(error);
+        },
+        "could not save. please try again.");
+    });
+
+    $(".BtnRemoveToken").click(function(e) {
+      e.preventDefault();
+
+      var $this = $(this);
+
+      var url = $this.data("url");
+      var postVariable = {};
+      postVariable.tokenId = $this.data("tokenid").toString();
+
+      utils.JSend(url, postVariable, function(data) {
+          $(".ApiTokensFeedback").showSuccess("Token removed");
+          $this.closest(".js-token").fadeOut();
+        }, function(error) {
+          $(".ApiTokensFeedback").showError(error);
+        },
+        "Could not remove token, please try again.");
+    });
+
+
+    //Notifications
+
+
+    $(".jsDropdownEmail a").click(function(e) {
+      e.preventDefault();
+
+      var postVariable = {};
+      postVariable.email = $(this).text();
+
+      var url = $(".jsDropdownEmail").data("url");
+      utils.post(url, postVariable, function(e) {
+        if (e.error) {
+          $(".jsMainEmailFeedback").showError(e.error);
+        } else {
+          $('.jsSelectedMainEmail').text(postVariable.email);
+          $(".jsMainEmailFeedback").showSuccess("Saved");
+        }
+      }, function() {
+        $(".jsMainEmailFeedback").showError("Could not save. Please try again.");
+      });
+    });
+
+    $(".jsAccountEmail").change(function() {
+      var postVariable = {};
+      postVariable.emailNotification = $(this).prop('checked') ? true : false;
+
+      var url = $(this).data("url");
+      var feedback = $(this).closest('tr').find('.jsEmailFeedback');
+      utils.JSend(url, postVariable, function(data) {
+          feedback.showSuccess("Saved");
+        }, function(error) {
+          feedback.showError(error);
+        },
+        "Could not save. Please try again.");
+      
+      utils.JSend(url, postVariable, function(data) {
+          feedback.showSuccess("Saved");
+        }, function(error) {
+          feedback.showError(error);
+        },
+        "Could not save. Please try again.");
+      
+      utils.JSend(url, postVariable, function(data) {
```

Comment:
![Codacy](https://app.dev.codacy.org/assets/images/favicon.png) Issue found: [A function with a name starting with an uppercase letter should only be used as a constructor. (new-cap)](https://app.dev.codacy.org/app/codacytestuser/CheckPrCommentsPublicProject/pullRequest?prid=8026)